### ISSUE TITLE
Real-time updates not working

### ISSUE THREAD

**Issue by kipusoep** [microsoft/vsts-extension-retrospectives#374]
It seems the current version (cloud) has issues with the real-time update.
Errors from the console:
```
reflect-bundle.js:2 
       [2022-07-05T09:08:43.999Z] Error: Connection disconnected with error 'Error: Server returned an error on close: Connection closed with an error.'. []
error @ reflect-bundle.js:2
log @ reflect-bundle.js:2
_stopConnection @ reflect-bundle.js:2
transport.onclose @ reflect-bundle.js:2
_close @ reflect-bundle.js:2
stop @ reflect-bundle.js:2
_stopInternal @ reflect-bundle.js:2
await in _stopInternal (async)
stop @ reflect-bundle.js:2
_stopInternal @ reflect-bundle.js:2
_processIncomingData @ reflect-bundle.js:2
L.connection.onreceive @ reflect-bundle.js:2
r.onmessage @ reflect-bundle.js:2

reflect-bundle.js:2         
       SignalR closed with an error of:  [Error: Server returned an error on close: Connection closed with an error.
    at L._processIncomin…]
```

**Comment by polatengin** [comment]
hi @kipusoep , @KyleStay ,

there shouldn't be this issue. may I kindly ask you to use a different browser, or, use the browser in incognito mode, please?

if all fails, how can I reach to you and have a troubleshooting session, maybe?

**Comment by kipusoep** [comment]
I'll try tomorrow, but all of my colleague had the same issue.
If it still doesn't work I will let you know and we'll schedule a session.

**Comment by polatengin** [comment]
@kipusoep yeah, that'd be awesome! I'm checking the "home page" of the retro extension, and I don't see any issue saying there is a connection problem to the sync service 🤔

**Comment by KyleStay** [comment]
@polatengin This is definitely broken for me and the rest of my team still. Looks like a CSP, browser (edge) enforced issue from what the console is saying. Also it appears to be wider spread per the reactions to my comment above. Please investigate.

**Comment by polatengin** [comment]
hi @kipusoep , @KyleStay , these issues must be resolved, I just released a new version with a few fixes (v1.80.4)

please feel free to re-open this issue and let me know if you see any problem with the sync service

**Comment by sbingham-MET** [comment]
Our ADO Services org is also having issue with the live sync, noticed it was not working this morning, just like others, since retros are every two weeks not sure exactly when it broke within the last 14 days.
